app/styles/ember-power-select-infinity.scss

```diff
@@ -9,13 +9,19 @@
 }
 
 .ember-power-select-infinity {
+    &-clear-btn {
+        right: 25px;
+        top: 4px;
+        position: absolute;
+    }
     &-trigger {
         padding: 0;
         border: none;
     }
     &-loading-indicator {
-        right: 5px;
-        top: 5px;
+        right: 25px;
+        top: 50%;
+        transform: translateY(-50%);
```

Comment:
`margin-top: -10px;` would be a bit cleaner than using a `transform` here, so you dont get an unnecessary accelerated layer.